src/PrestaShopBundle/Resources/config/services/adapter/order.yml

```diff
@@ -133,3 +133,13 @@ services:
     tags:
       - name: tactician.handler
         command: 'PrestaShop\PrestaShop\Core\Domain\Order\Query\GetOrderPreview'
+
+  prestashop.adapter.order.query_handler.search_products_for_order_creation:
+    class: PrestaShop\PrestaShop\Adapter\Order\QueryHandler\SearchProductsForOrderCreationHandler
+    arguments:
+      - '@prestashop.adapter.legacy.context'
+      - '@prestashop.core.localization.locale.repository'
```

Comment:
You should use `prestashop.core.localization.locale.context_locale` here

Reply:
maybe you know what is going on with it? I mean that Locale implements LocaleInterface which has no methods.

Reply:
I'm sorry, I did not get the question